src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/ErrorPage.css

```diff
@@ -168,3 +152,42 @@ a {
         color: #13709e;
         text-decoration: underline;
     }
+
+.showRawException {
+    font-size: 1.2em;
+    cursor: pointer;
```

Comment:
The issue seems to happen if I use an anchor tag without the `href` attribute. Was reading on the web that tab order is one of the side effects of removing this attribute. Having a value like # for the href would result in the url being modified which i think we might not prefer, so probably button is a better approach here.